src/servers/express/index.ts

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+import mongoose from 'mongoose';
+import express, { Express, Router } from 'express';
+import cors from 'cors';
+import AdminJS from 'adminjs';
+import { ADMIN, createAdmin, generateAdminJSConfig } from '../../admin';
+import { expressAuthenticatedRouter } from '../../admin/router';
+import { init } from '../../sources/mikroorm/config';
+import dataSource from '../../sources/typeorm/config';
+import { ViewController } from './views/view.controller';
+
+const getAdminRelativePath = (admin: AdminJS, path: string) => {
+  const { rootPath } = admin.options;
+
+  return path.replace(rootPath, '');
+};
+
+const attachAdminJS = async (app: Express) => {
+  const config = generateAdminJSConfig();
+  const adminJS = new AdminJS(config);
+
+  const adminPredefinedRouter = Router();
+  const viewController = new ViewController(adminJS);
+  adminPredefinedRouter.get(
```

Comment:
predefined router is required because you cannot override express routes that are already created, this way we can add `GET /login` and `buildAuthenticatedRouter` will later add `POST /login` and other routes to the existing router